packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -81,6 +82,15 @@ export const attachmentUploadCardsStyles = mergeStyles({
   overflow: 'auto'
 });
 
+/**
+ * @private
+ */
+export const useCustomStyles = makeStyles({
+  root: {
+    backgroundColor: 'rgb(0,0,0,0)'
```

Comment:
```suggestion
    backgroundColor: 'transparent'
```
In this case we will be consistent with other places where we use transparent background and it's a little bit more readable

Reply:
thanks updated